pkgs/applications/misc/plasma-theme-switcher/default.nix

```
@@ -0,0 +1,37 @@
+{
+  stdenv, lib, cmake, extra-cmake-modules, fetchFromGitHub, qtbase, kdeFrameworks
+}:
+
+stdenv.mkDerivation rec {
+  pname = "plasma-theme-switcher";
+  version = "0.1";
+  dontWrapQtApps = true;
+
+  src = fetchFromGitHub {
+    owner = "maldoinc";
+    repo = "plasma-theme-switcher";
+    rev = "v${version}";
+    sha256 = "sdcJ6K5QmglJEDIEl4sd8x7DuCPCqMHRxdYbcToM46Q=";
+  };
+
+  buildInputs = [ qtbase kdeFrameworks.plasma-framework ];
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=" ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp plasma-theme $out/bin
```

```suggestion
    runHook preInstall

    mkdir -p $out/bin
    install -Dm755 ./plasma-theme $out/bin/plasma-theme
    
    runHook postInstall
```

Also, is there a reason why we can't use cmake's install procedure?

---

The source code does not provide an install target. Leaving the override out will cause the installation to fail on this package